server/src/controllers/Auth.controller.js

```diff
@@ -0,0 +1,66 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import User from '../models/User.model';
+import dummyData from '../dummyDb/db';
+
+dotenv.config();
+
+const { users } = dummyData;
+
+/**
+ * @class AuthController
+ */
+export default class AuthController {
+  /**
+   *@method signup
```

Comment:
Please leave a space between * and @method. Also always leave new lines between JSDoc categories to aid readability